src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task{
```

Comment:
There should be whitespace within a statement, so consider "public class Event extends Task {", similarly for line 6, 9 and 13.